client/templates/components/get_certificate.html

```diff
@@ -0,0 +1,4 @@
+<h3>תג זה הינו הכרחי עבור ביצוע ההתנדבות - שמור את התג בטלפון שלך</h3>
+<img src="{{ certificate.image_url }}" alt="volunteer tag" width="400" />
+<br />
+<a href="{% url 'download_certificate' pk=certificate.id %}" download>הורדת תג</a>
```

Comment:
is it used more then once? otherwise merge it into thanks_volunteer

Reply:
Yes, in thanks_volunteer and find_volunteer